static/styles/codemirror.css

```diff
@@ -1,341 +0,0 @@
-/* BASICS */
```

Comment:
Do we need to remove references to these css files from any HTML files?

Reply:
No, we don't need to. The HTML files use the copies stored in static/css-vendor.

Eventually, I'd like webpack to bundle the css files.